sway/input/cursor.c

```diff
@@ -26,6 +26,7 @@
 #include "sway/tree/view.h"
 #include "sway/tree/workspace.h"
 #include "wlr-layer-shell-unstable-v1-protocol.h"
+#include "libtouch.h"
```

Comment:
This is an external dependency, so angle brackets should be used (`#include <libtouch.h>`).